Library/RemoteConfig/RemoteConfigFeature+HelpersTests.swift

```diff
@@ -3,111 +3,118 @@ import Prelude
 import XCTest
 
 final class RemoteConfigFeatureHelpersTests: TestCase {
-  func testBlockUsers_RemoteConfig_FeatureFlag_False() {
+  func assertFeatureIsFalse(
+    whenRemoteConfigFeatureIsFalse feature: RemoteConfigFeature,
+    checkFeatureFlag: () -> Bool
+  ) {
     let mockRemoteConfigClient = MockRemoteConfigClient()
-      |> \.features .~ [RemoteConfigFeature.blockUsersEnabled.rawValue: false]
+      |> \.features .~ [feature.rawValue: false]
 
     withEnvironment(remoteConfigClient: mockRemoteConfigClient) {
-      XCTAssertFalse(featureBlockUsersEnabled())
+      XCTAssertFalse(checkFeatureFlag())
     }
   }
 
-  func testBlockUsers_RemoteConfig_FeatureFlag_True() {
+  func assertFeatureIsTrue(
+    whenRemoteConfigFeatureIsTrue feature: RemoteConfigFeature,
+    checkFeatureFlag: () -> Bool
+  ) {
     let mockRemoteConfigClient = MockRemoteConfigClient()
-      |> \.features .~ [RemoteConfigFeature.blockUsersEnabled.rawValue: true]
+      |> \.features .~ [feature.rawValue: true]
 
     withEnvironment(remoteConfigClient: mockRemoteConfigClient) {
-      XCTAssertTrue(featureBlockUsersEnabled())
+      XCTAssertTrue(checkFeatureFlag())
```

Comment:
😅 Great catch @ifosli. Soon as I swapped this to `XCTAssertTrue` the tests started failing!

Reply:
Looks good now!